crates/pixi_config/src/lib.rs

```diff
@@ -461,6 +461,7 @@ impl From<&Config> for rattler_repodata_gateway::ChannelConfig {
                 jlap_enabled: !config.disable_jlap.unwrap_or(false),
                 zstd_enabled: !config.disable_zstd.unwrap_or(false),
                 bz2_enabled: !config.disable_bzip2.unwrap_or(false),
+                sharded_enabled: false,
```

Comment:
These changes are available from the `main` branch! Maybe you can also merge that in here?